CalibPPS/TimingCalibration/plugins/PPSTimingCalibrationPCLWorker.cc

```diff
@@ -41,7 +41,14 @@ class PPSTimingCalibrationPCLWorker : public DQMGlobalEDAnalyzer<TimingCalibrati
                       const edm::EventSetup&,
                       TimingCalibrationHistograms&) const override;
 
-  edm::EDGetTokenT<edm::DetSetVector<CTPPSDiamondRecHit>> diamondRecHitToken_;
+  template <typename T>
+  bool searchForProduct(edm::Event const& iEvent,
+                        const std::vector<edm::EDGetTokenT<T>>& tokens,
+                        const std::vector<edm::InputTag>& tags,
+                        edm::Handle<T>& handle) const;
+
+  std::vector<edm::InputTag> RecHitTags_;
+  std::vector<edm::EDGetTokenT<edm::DetSetVector<CTPPSDiamondRecHit>>> diamondRecHitTokens_;
   edm::ESGetToken<CTPPSGeometry, VeryForwardRealGeometryRecord> geomEsToken_;
```

Comment:
I think the ESGetToken for sure can be const :)

Reply:
Yes :) and the `tags` variable too